### ISSUE TITLE
RFE: Allow Ansible playbooks stored within a collection to be defined within a orchestrate action

### ISSUE THREAD

**Issue by ryankwilliams** [RedHatQE/teflo#131]
Today teflo allows users to be able to create an orchestrate action that has the following:
* Orchestrate = Ansible
* Ansible playbook = my_playbook.yml (where the playbook is stored within the repository where the teflo SDF file is)

With the addition of Ansible collections, collections now allows you to be able to store playbooks inside a collection. Making it very easy to call the playbook (using dot notation) once the collection is installed.

```
# From Ansible documentation
ansible-playbook my_namespace.my_collection.playbook1 -i ./myinventory
```

https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#using-collections-in-playbooks

This feature request is for teflo to allow the user to be able to define playbooks in their orchestrate action where the playbook is stored in a collection (which teflo would install). Right now this is not possible as validation fails due to the playbook needs to be on the file system in order for teflo to properly find it.

```
---
name: orchestrate
description: Install OpenShift

orchestrate:
  - name: Install OpenShift
    orchestrator: ansible
    hosts: hypervisor
    ansible_playbook:
      name: ansible.collection.install_openshift <<<<
    ansible_options:
      extra_vars:
        disconnected_install: "{{ DISCONNECTED }}"
```

Without this, the workaround is to create a playbook within the repository where the SDF file is and then have that playbook call the playbook stored in the collection using the `import_playbook` module.

```
---
name: orchestrate
description: Install OpenShift

orchestrate:
  - name: Install OpenShift
    orchestrator: ansible
    hosts: hypervisor
    ansible_playbook:
      name: ansible/install_openshift.yml <<<<
    ansible_options:
      extra_vars:
        disconnected_install: "{{ DISCONNECTED }}"
```

```
---
- import_playbook: ansible_collection.install_openshift
```

**Comment by dno-github** [comment]
Jira ticket CCITCARBON-369 has been created for this issue. Url : https://issues.redhat.com/browse/CCITCARBON-369

**Comment by JunqiZhang0** [comment]
Working on this

**Comment by ryankwilliams** [comment]
Please find the attached archive with an example to test this issue.

Steps to use it.

1. Untar the file `tar xvf teflo-131.tar.gz`
2. Run run.sh `bash run.sh`

[teflo-131.tar.gz](https://github.com/RedHatQE/teflo/files/8217059/teflo-131.tar.gz)

**Comment by rujutashinde** [comment]
fixed by #190
